packages/rs-drive-abci/tests/strategy_tests/main.rs

```diff
@@ -2015,6 +2017,7 @@ mod tests {
         let block_count = 120;
         let mut platform = TestPlatformBuilder::new()
             .with_config(config.clone())
+            .with_initial_protocol_version(PROTOCOL_VERSION_1)
```

Comment:
this test shouldn't need to set this.

Reply:
Same https://github.com/dashpay/platform/pull/2206#discussion_r1787753976